deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -4,10 +4,17 @@
 require_relative 'app_screenshot'
 require_relative 'module'
 require_relative 'loader'
+require_relative 'queue_worker'
+require_relative 'app_screenshot_iterator'
 
 module Deliver
   # upload screenshots to App Store Connect
   class UploadScreenshots
+    DeleteScreenshotJob = Struct.new(:app_screenshot, :localization, :app_screenshot_set)
+    UploadScreenshotJob = Struct.new(:app_screenshot_set, :path, :wait_for_processing)
+
+    NUMBER_OF_THREADS = Helper.test? ? 1 : 10
```

Comment:
`10` is just the number that an app screenshot set has as its limit. So it won't go crazy more than what App Store Connect would be capable of.